components/button/index.en-US.md

```diff
@@ -1,8 +1,34 @@
 ---
 category: Components
-chinese: 按钮
-type: 基本
-english: Button
+type: Basic
+title: Button
 ---
 
-TBD
+To trigger an operation.
+
+## Occasion
+
+A button means an operation(or a series of operations). Click a button will trigger corresponding business logic.
+
+## API
+
+To get a customized button, just set `type`/`shape`/`size`/`loading`/`disabled`.
+
+Property | Description | Type | Default
+-----|-----|-----|------
+type | can be set as `primary` `ghost` or nothing | string | -
+htmlType | to set the original `type` of `button`, see: [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-type) | string | `button`
+icon | set the icon of button, see: Icon component | string | -
+shape | can be set as `circle` `circle-outline` or nothing | string | -
+size | can be set as `small` `large` or nothing | string | `default`
+loading | to set the loading status of button | boolean | false
+onClick | set the handler to handle `click` event | function | -
+
+`<Button>Hello world!</Button>` will be rendered into `<button>Hello world!</button>`, and all the properties which are not listed above will be transfered to `<button />`.
```

Comment:
... and all the properties which are not listed above will be transferred to the `<button>` tag.